StarGeometry compilation fail

PR #202 broke the DEV [AutoBuild](https://www.star.bnl.gov/public/comp/prod/Sanity/AutoBuild.html) tonight:
```
.sl73_gcc485/OBJ/StarVMC/StarGeometry/BbcmGeo.cxx: In member function 'virtual void BBCMGEO::BPOL::Block(AgCreate)':
.sl73_gcc485/OBJ/StarVMC/StarGeometry/BbcmGeo.cxx:516:33: error: 'class AgModule' has no member named 'AddCut'
                       module()-&gt;AddCut(active()-&gt;GetName(),"cutgam",0.00008);           
                                 ^
.sl73_gcc485/OBJ/StarVMC/StarGeometry/BbcmGeo.cxx:517:33: error: 'class AgModule' has no member named 'AddCut'
                       module()-&gt;AddCut(active()-&gt;GetName(),"cutele",0.001);           
                                 ^
...
```

Actually, it wasn't exactly the same error this time. Instead of "AddCut", it's complaining about "AddHit" over and over now....

/afs/rhic.bnl.gov/star/packages/gitdev/.sl73_gcc485/obj/StarVMC/StarGeometry/EiddGeo.cxx: In member function 'virtual void EIDDGEO::TGAP::Block(AgCreate)':
/afs/rhic.bnl.gov/star/packages/gitdev/.sl73_gcc485/obj/StarVMC/StarGeometry/EiddGeo.cxx:792:33: error: 'class AgModule' has no member named 'AddHit'
                       module()-&gt;AddHit( "TGAP", "z", 0.0005, 0, 0, "s");           
                                 ^
/afs/rhic.bnl.gov/star/packages/gitdev/.sl73_gcc485/obj/StarVMC/StarGeometry/EiddGeo.cxx:793:33: error: 'class AgModule' has no member named 'AddHit'
                       module()-&gt;AddHit( "TGAP", "y", 0.0005, 0, 0, "");           
                                 ^
...

--- a/mgr/Conscript-standard
+++ b/mgr/Conscript-standard
@@ -386,12 +386,19 @@ if ( $pkg !~ /^sim$/ && $pkg !~ /^gen$/ ) {
 			# generally enable all warnings for now
 			$env->{ENV}->{AGML_WARNINGS} = 1==1; # $param::debug;
 
+			# cons doesn't come with a scanner for python dependencies
+			use File::Find;
+			my @agml_deps;
+			find(sub { return unless -f; return unless /\.py$/; push @agml_deps, "#$File::Find::name"; }, "mgr/Dyson");
+			push @agml_deps, "#$xml";
+			push @agml_deps, "#$agml_cmd";
+
 			if ( $xml =~ m/Compat/ ){
-			    Depends $env ["$OBJ/$agE"], ("#$xml", "#$agml_cmd");
+			    Depends $env ["$OBJ/$agE"], @agml_deps;
 			    Command $env ["$OBJ/$agE"], ("#$xml"), qq($agml_cmd --file=%1 --module=$agM --export=Mortran > %>);
 
 			} else {
-			    Depends $env ["$OBJ/$agX", "$OBJ/$agH", "$OBJ/$agE"], ("#$xml", "#$agml_cmd");
+			    Depends $env ["$OBJ/$agX", "$OBJ/$agH", "$OBJ/$agE"], @agml_deps;
 			    # Depends $env --> some rule to build extra dict on the fly
 
 			    Command $env ["$OBJ/$agX", "$OBJ/$agH"], ("#$xml"), qq($agml_cmd --file=%1 --module=$agM --export=AgROOT --path=%>:d);

--- a/mgr/Conscript-standard
+++ b/mgr/Conscript-standard
@@ -369,8 +369,9 @@ if ( $pkg !~ /^sim$/ && $pkg !~ /^gen$/ ) {
 		foreach (@ODIR){ if ( ! -d $ODIR[0] ){  mkdir($ODIR[0],0755);}}
 
 		# local or global install
-		if ( -e "mgr/agmlParser.py"){	$agml_cmd = "mgr/agmlParser.py";
-		} else {			$agml_cmd = $env->{ENV}->{STAR}."/mgr/agmlParser.py";}
+		if ( -e "mgr/agmlParser.py"){	$agml_root = "mgr";
+		} else {			$agml_root = $env->{ENV}->{STAR}."/mgr";}
+		$agml_cmd = "$agml_root/agmlParser.py";
 
 		foreach $xml (@xmlGeo){
 		    if ( $xml =~ /StarVMC\/Geometry\// ){
@@ -389,7 +390,7 @@ if ( $pkg !~ /^sim$/ && $pkg !~ /^gen$/ ) {
 			# cons doesn't come with a scanner for python dependencies
 			use File::Find;
 			my @agml_deps;
-			find(sub { return unless -f; return unless /\.py$/; push @agml_deps, "#$File::Find::name"; }, "mgr/Dyson");
+			find(sub { return unless -f; return unless /\.py$/; push @agml_deps, "#$File::Find::name"; }, "$agml_root/Dyson");
 			push @agml_deps, "#$xml";
 			push @agml_deps, "#$agml_cmd";
 

